registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/exception/util/PlatformErrorMessages.java

```diff
@@ -1401,6 +1401,19 @@ public enum PlatformErrorMessages {
 	/** The queue json validation failed. */
 	QUEUE_JSON_VALIDATION_FAILED(PlatformConstants.RPR_MA_MIDDLEWARE + "004",
 			"Unable to find Connection Properties"),
+	
+	RPR_MAS_MISSING_INPUT_PARAMETER(PlatformConstants.RPR_USER_NOTIFICATION_API + "001",
```

Comment:
Please change MSA to UNA in all error codes

Reply:
done, please check earliest commit